src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -681,10 +681,10 @@ def _build_app_remote(mode, src_dir, publish=False, destination_override=None,
         shutil.rmtree(temp_dir)
 
 
-def build_app_from(applet_id, version, publish=False, do_try_update=True, bill_to_override=None,
+def build_app_from(applet_desc, version, publish=False, do_try_update=True, bill_to_override=None,
```

Comment:
We are injecting `args._from` still but now it is the description of the applet.